install/kubernetes/helm/skywalking/README.md

```diff
@@ -0,0 +1,231 @@
+| `elasticsearch.data.persistence.name` | `data` | Data statefulset PVC template name |
+| `elasticsearch.data.persistence.size` | `30Gi` | Data persistent volume size |
+| `elasticsearch.data.persistence.storageClass` | `nil` | Data persistent volume Class |
+| `elasticsearch.data.persistence.accessMode` | `ReadWriteOnce` | Data persistent Access Mode |
+| `elasticsearch.data.readinessProbe` | see `values.yaml` for defaults | Readiness probes for data-containers |
+| `elasticsearch.data.podAnnotations` | `{}` | Data StatefulSet annotations |
+| `elasticsearch.data.nodeSelector` | `{}` | Node labels for data pod assignment |
+| `elasticsearch.data.tolerations` | `[]` | Data tolerations |
+| `elasticsearch.data.terminationGracePeriodSeconds` | `3600` | Data termination grace period (seconds) |
+| `elasticsearch.data.antiAffinity` | `soft` | Data anti-affinity policy |
+| `elasticsearch.data.nodeAffinity` | `{}` | Data node affinity policy |
+| `elasticsearch.data.podManagementPolicy` | `OrderedReady` | Data pod creation strategy |
+| `elasticsearch.data.updateStrategy` | `{type: "onDelete"}` | Data node update strategy policy |
+
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```console
+$ helm install myrelease skywalking --set nameOverride=newSkywalking
+```
+
+Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the chart. For example,
+
+```console
+$ helm install my-release skywalking -f values.yaml
+```
+
+> **Tip**: You can use the default [values.yaml](values.yaml)
+
+### RBAC Configuration
+Roles and RoleBindings resources will be created automatically for `OAP` .
+
+> **Tip**: You can refer to the default `oap-role.yaml` file in [templates](templates/) to customize your own.
+
+### Ingress TLS
+If your cluster allows automatic create/retrieve of TLS certificates (e.g. [kube-lego](https://github.com/jetstack/kube-lego)), please refer to the documentation for that mechanism.
+
+To manually configure TLS, first create/retrieve a key & certificate pair for the address(skywalking ui) you wish to protect. Then create a TLS secret in the namespace:
+
+```console
+kubectl create secret tls skywalking-tls --cert=path/to/tls.cert --key=path/to/tls.key
+```
+
+Include the secret's name, along with the desired hostnames, in the skywalking-ui Ingress TLS section of your custom `values.yaml` file:
+
+```yaml
+ui:
+  ingress:
+    ## If true, Skywalking ui server Ingress will be created
+    ##
+    enabled: true
+
+    ## Skywalking ui server Ingress hostnames
+    ## Must be provided if Ingress is enabled
+    ##
+    hosts:
+      - skywalking.domain.com
+
+    ## Skywalking ui server Ingress TLS configuration
+    ## Secrets must be manually created in the namespace
+    ##
+    tls:
+      - secretName: skywalking-tls
+        hosts:
+          - skywalking.domain.com
+```
+### Envoy ALS
```

Comment:
This should be linked in https://github.com/apache/skywalking/blob/master/docs/en/setup/envoy/als_setting.md doc too.